Chore: Document making the text icon bold with t-class in lists

kaixin-hc opened the pull request:
**What is the purpose of this pull request?**

- [X] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

Addresses #2489 
<!--
  If this pull request is addressing an issue, link to the issue: #xxx

  If this pull request completely addresses an issue, use one of the closing keywords: "Fixes #xxx" or "Resolves #xxx"
  https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

  Otherwise, elaborate further on the rationale of this pull request as needed
-->

**Overview of changes:**
Adds an example of bold text to the customising text-icon for bold text section

**Anything you'd like to highlight/discuss:**
I don't think the look of it is great but I think this makes it visible for users. 
<img width="836" alt="Screenshot 2024-04-28 at 11 14 31 AM" src="https://github.com/MarkBind/markbind/assets/59471320/1ecb1b8b-af04-4d9f-837e-c492b64bc104">


**Testing instructions:**

<!--
  Any special testing instructions **not including** our automated tests.
-->

**Proposed commit message: (wrap lines at 72 characters)**
Chore: document making the text icon bold

<!--
  See this link for more info on how to write a good commit message:
  https://se-education.org/guides/conventions/git.html

  As best as possible, write a succinct commit title in 50 characters

|---------This is the width of 50 chars----------|
|-----------This is the width of 72 chars for your reference-----------|
-->

---

**Checklist:** :ballot_box_with_check:

<!-- Leave non-applicable items unchecked -->

- [X] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [ ] Linked all related issues
- [X] No unrelated changes <!-- It's tempting, but increases the reviewer's work, and really pollutes the commit history =( -->

<!--
  We'll try our best to get to your PR within a week.
  If we haven't gotten to it then, or if your pull request resolves an urgent item, feel free to give us a ping!
-->

---

**Reviewer checklist:**

Indicate the [SEMVER](https://semver.org/) impact of the PR:
- [ ] Major (when you make incompatible API changes)
- [ ] Minor (when you add functionality in a backward compatible manner)
- [ ] Patch (when you make backward compatible bug fixes)

At the end of the review, please label the PR with the appropriate label: `r.Major`, `r.Minor`, `r.Patch`.

Breaking change release note preparation (if applicable):
- To be included in the release note for any feature that is made obsolete/breaking

> Give a brief explanation note about:
>   - what was the old feature that was made obsolete
>   - any replacement feature (if any), and
>   - how the author should modify his website to migrate from the old feature to the replacement feature (if possible).

EltonGohJH reviewed (commented):
I am okay with the change.  
I think I could not tell that the text is bolded if I am not looking for it.

Are there any areas where the doc mention that you can use `fw-bold` to bold? If not we should mention it somewhere on the docs. Wdyt

kaixin-hc commented:
You make a good point - I've added a tooltip that signposts to look at the bootstrap documentation, and also added italics to the one that is below so that the text having difference is more expected. See below picture

---
<img width="875" alt="Screenshot 2024-04-28 at 9 15 57 PM" src="https://github.com/MarkBind/markbind/assets/59471320/2b86e761-afd1-47a7-b779-6f065b6a1d03">

EltonGohJH reviewed (approved):
LGTM. Thanks for the change!
